packages/components/src/mobile/bottom-sheet/bottom-sheet-navigation/navigation-screen.native.js

```diff
@@ -128,7 +128,16 @@ const BottomSheetNavigationScreen = ( {
 				</TouchableHighlight>
 			</ScrollView>
 		);
-	}, [ children, isFocused, safeAreaBottomInset, listProps ] );
+	}, [
+		children,
+		isFocused,
+		safeAreaBottomInset,
+		listProps,
+		name,
+		isScrollable,
+		isNested,
+		onLayout,
+	] );
```

Comment:
I observed that we have another variable (`styles.scrollableContent.paddingBottom`) that is used within the memoized code that we might be added to the dependencies array, wdyt?

Reply:
For this case, the value of `paddingBottom` is fixed as it comes from the static styles file. Do you think we should add it anyways? It wouldn't affect having it or not in the dependencies list right?

Reply:
Oh, right, `paddingBottom` is a static value as it comes from `styles` object. In that case, it's ok not to include it 👍 .